### PR TITLE
README Instructions and Install Guide

### PR DESCRIPTION
**Pull request by Ompragash**
# Summary

- Updated README.md to help users get started with the `harness` cli utility. 
- Added `INSTALL.md` file with the install instructions.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Installing the Harness CLI by downloading the binary
+
+You can install the Harness CLI (`harness`) in order to interact with Harness Platform from a command-line interface. You can install `harness` on Linux, Windows, or macOS.
+
+<!-- Regenerate this table of contents using https://github.com/ekalinin/github-markdown-toc -->
+<!-- gh-md-toc --insert README.md -->
+<!--ts-->
```
Comment (devinder-harness) [inline INSTALL.md:7]: can we remove this ?

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```
+
+4. Move the `harness` binary in a directory that is on your `PATH`.
+    - To check your `PATH`, execute the following command:
+        ```bash
+        echo $PATH
+        ```
+    - To move the `harness` binary
+        ```bash
+        mv ./harness /PATH/TO/DEST
```
Comment (devinder-harness) [inline INSTALL.md:42]: we are moving away from this and moving more towards PATH setup 
```
tar -xvf harness-v0.0.13-alpha-darwin-amd64.tar.gz 
export PATH="$(pwd):$PATH" 
echo 'export PATH="'$(pwd)':$PATH"' >> ~/.bash_profile
```

File: INSTALL.md (hunk 1 above)
Comment (Ompragash) [reply INSTALL.md:7]: yup we can remove that

File: INSTALL.md (hunk 2 above)
Comment (Ompragash) [reply INSTALL.md:42]: cool

Comment (Ompragash) [reply INSTALL.md:33]: The steps I provided offer users the flexibility to choose the PATH where they prefer to locate the binary. However, as we initially outlined the recommended steps in the getting started wizard, I'll mirror those steps for now. Let's aim to establish a standardized approach for installation, utilizing a setup script via curl.

**Pull request merged**